connect/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -189,7 +189,7 @@ object SparkConnectService {
    */
   def startGRPCService(): Unit = {
     val debugMode = SparkEnv.get.conf.getBoolean("spark.connect.grpc.debug.enabled", true)
-    val port = 15002
+    val port = SparkEnv.get.conf.getInt("spark.connect.grpc.binding.port", 15002)
```

Comment:
shall we call `SparkEnv.get.conf.get(CONNECT_GRPC_BINDING_PORT)`?

Reply:
Makes sense! Done!